xml/art_sle_amazon_ecs.xml

```diff
@@ -41,21 +41,27 @@
     <para>
       Creating a new ECS cluster in Amazon is accomplished by using the
       <link xlink:href="https://console.aws.amazon.com/ecs/home#/firstRun">First
-      Run Wizard</link>
+      Run Wizard.</link> Alternatively you may use the
+      <emphasis role="strong">aws</emphasis> command line tools to
```

Comment:
"... the aws command line tool**s**" ?

Is it really plural? From my perspective and as far as I can see, there is only one `aws` command (singular!) but with several subcommands (plural!).
Probably a matter of definition, just stumbled upon.